desktop-exporter/app/components/waterfall-view/duration-bar.tsx

```diff
@@ -51,12 +59,41 @@ export function DurationBar(props: DurationBarProps) {
   }
 
   let label = getDurationString(spanEndTimeNs - spanStartTimeNs);
+
+  let eventCircles = props.spanData.events.map((eventData, index) => {
+    let eventName = eventData.name;
+    let eventTimeNs = getNsFromString(eventData.timestamp);
+    let spanDurationNS = spanEndTimeNs - spanStartTimeNs;
+    let eventOffsetPercent = Math.floor(
+      ((eventTimeNs - spanStartTimeNs) / spanDurationNS) * 100,
+    );
+
+    return (
+      <li key={index}>
```

Comment:
`key={index}` is a bit of an anti-pattern since a new list with different values would have the same keys. Can you pull out an id here? or `${eventName}-${eventTimeNs}`?